File: z-clients/javascript/src/models/idempotencyConfigureNamespaceIn.ts
// this file is @generated

export interface IdempotencyConfigureNamespaceIn {
    name: string;
}

export const IdempotencyConfigureNamespaceInSerializer = {
    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _fromJsonObject(object: any): IdempotencyConfigureNamespaceIn {
        return {
            name: object['name'],
        };
    },

    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _toJsonObject(self: IdempotencyConfigureNamespaceIn): any {
        return {
            'name': self.name,
        };
    }
}

File: z-clients/rust/src/models/idempotency_configure_namespace_in.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct IdempotencyConfigureNamespaceIn {
    pub name: String,
}

impl IdempotencyConfigureNamespaceIn {
    pub fn new(name: String) -> Self {
        Self { name }
    }
}
